staging.gen3.biodatacatalyst.nhlbi.nih.gov/dashboard/Secure/reports/modules/lit-html/package.json

```diff
@@ -2,7 +2,7 @@
   "_args": [
     [
       "lit-html@1.3.0",
-      "/home/reuben/Code/PlanX/cloud-automation/files/dashboard/usage-reports"
+      "/home/bdcatprod/cloud-automation/files/dashboard/usage-reports"
```

Comment:
Oh god.. full paths instead of relative paths?
cc: @emalinowski is this kosher?

Reply:
I think Reuben set this up, and I don't have a lot of knowledge around the intricacies of the service. However, to me, this looks like a path on the adminvm, so you might just use, stagingdatastage for now, or we can see if we can use $GEN3_HOME instead of /home/bdcatprod/cloud-automation.